File: Lista 01 - Java /Ex19/src/ex19/Ex19.java
package ex19;

import java.util.Scanner;

/**
 * @author bryan
 */
public class Ex19 {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        double numero;
        int pisoNumero;
        int tetoNumero;

        System.out.print("Digite um número real: ");
        numero = sc.nextDouble();

        pisoNumero = Piso(numero);
        tetoNumero = Teto(numero);

        System.out.println("Piso: " + pisoNumero);
        System.out.println("Teto: " + tetoNumero);

    }

    public static int Piso(double numero) {
        return (int) numero;
    }

    public static int Teto(double numero) {

        if ((int) numero != numero) {
            return (int) (numero + 1);
        } else {
            return (int) numero;
        }

    }
}

File: Prática/Lista 04 - C/Ex19/Ex19.c
#include "../Console.c"
#include <stdio.h>
#include <stdlib.h>

int piso(float valor)
{
    return (int)valor;
}

int teto(float valor)
{
    if (valor == (int)valor)
    {
        return (int)valor;
    }
    else
    {
        return (int)valor + 1;
    }
}

int main()
{
    float valor = 5.3;

    printf("O piso do valor %g é: %d\n", valor, piso(valor));
    printf("O teto do valor %g é: %d\n", valor, teto(valor));

    return 0;
}
